### DISCUSSION TITLE
Docker update - lost my data

### DISCUSSION THREAD

**Discussion by erikgadioli** [netbox-community/netbox-docker#903] [Getting Help] [answered]
Hi all,

Happily using a containerized version of Netbox, witch could use a update. I was running 3.3.9 and 3.4.1 was out.
It was not running in detached mode so i just did a crl-z and forgot about de docker compose down.
Then i did:
- git checkout release && git pull -p origin release
- docker-compose pull
- docker-compose up
This did not work well. something with "contains an invalid type, it should be an array" 
I guesed this was because of an old version of docker-compose. On ubuntu 20 docker-compose 1.25 is the latest. So i removed the old one, apt remove docker-compose. along with some not needed dependancies.
after this i did a curl to get the latest docker compose 2.14.2, and a chmod -x.
- had heath check issues
- had layer issues, docker: failed to register layer - moved the file
- Postgres issues "The data directory was initialized by PostgreSQL version 14, which is not compatible with this version 15.1"
   i did a docker-compose down -v

now i have a now instance of netbox i think. Any change my old data is anywhere? 

clearly i am no expert and i am trying fixing things not knowing what i am doing. Hope you have some ideas.

 Thanks 
EG

**Comment by tobiasge** [comment, marked as answer]
With `docker-compose down -v` you deleted all volumes. If you didn't take a backup of your database then the old data is gone.

**Reply by erikgadioli** [reply]
thank you tobias. i have some work to do....uhum...
